src/middlewares/getOneValidation.js

```diff
@@ -0,0 +1,16 @@
+import Joi from 'joi';
+
+export const schema = Joi.object({
+    name: Joi.string().required(),
+    street: Joi.required(),
+    postalCode: Joi.string().required(),
+});
+
+export const getOneValidation = (req, res, next) => {
+    const result = schema.validate(req.query);
+    if (result.error) {
+        next(new Error(result.error.details[0].message));
```

Comment:
what if `result.error.details` has no element? this code will crash.
better to use lodash get here for more defensive coding.

```suggestion
        const errorMessage = lodashGet(result, 'error.details[0].message', 'Generic error message');
        next(new Error(errorMessage));
```

Reply:
Wow Thank you so much :)